gateway/api/views/jobs.py

```diff
@@ -249,6 +249,73 @@ def result(self, request, pk=None):  # pylint: disable=invalid-name,unused-argum
             serializer = self.get_serializer(job)
         return Response(serializer.data)
 
+    @action(methods=["POST"], detail=True)
```

Comment:
Let's start using "PATCH" for those methods that will update a existed entry in the DB (we will see how to manage the migration for other end-points).